psa-crypto/src/operations/other.rs

```diff
@@ -2,3 +2,33 @@
 // SPDX-License-Identifier: Apache-2.0
 
 //! # Other cryptographic services
+//!
+//! See the PSA Crypto API for the format of the different parameters used in this module.
+
+use crate::initialized;
+use crate::types::status::{Result, Status};
+
+/// Generate a buffer of random bytes.
+///
+/// The random bytes are written into `output`. The function returns the number of random bytes
+/// generated.
```

Comment:
Is it necessary to return the number of bytes generated? The function could just say in the documentation here that it will generate as many bytes as there is space in the `output` slice.

Reply:
Not really -- I can change this, 2 secs :)